NitroxClient/Logger/ClientLogger.cs

```diff
@@ -0,0 +1,44 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace NitroxClient.Logger
+{
+    public class ClientLogger
+    {
+        public const int LOG_CONSOLE = 1;
+        public const int LOG_ERRORMESSAGE = 2;
+        public const int LOG_DEBUG = 4;
+        private static int logLocations = LOG_ERRORMESSAGE;
+
+        // Set with combinaition of constants -- setLogLocation(LOG_CONSOLE | LOG_ERRORMESSAGE)
```

Comment:
Honestly, I feel that bitmasking is a little weird in high level languages.  It is not immediately obvious and this is not an environment with space constraints.  It also makes for weird code throughout the implementation, ie: if ((logLocations & LOG_ERRORMESSAGE) != 0).  

Personally, I think a better implementation would be a LogLevel enumeration(None, Debug, Error) and a boolean for enabling console logging.